python_modules/airline-demo/airline_demo/types.py

```diff
@@ -24,12 +24,14 @@ class SparkDataFrameS3StoragePlugin(TypeStoragePlugin):  # pylint: disable=no-in
     @classmethod
     def set_object(cls, object_store, obj, _context, _runtime_type, paths):
         target_path = object_store.key_for_paths(paths)
-        obj.write.parquet('s3a://' + target_path)
+        obj.write.parquet('s3a://' + object_store.bucket + '/' + target_path)
```

Comment:
may be nice to have helper method to generate s3 paths rather than the minor code dup

Reply:
yup